test/e2e/windows/host_process.go

```diff
@@ -154,6 +155,187 @@ var _ = SIGDescribe("[Feature:WindowsHostProcessContainers] [Excluded:WindowsDoc
+			},
+			{
+				command:    []string{"uname.exe", "-o"},
+				workingDir: "%CONTAINER_SANDBOX_MOUNT_POINT%/bin/",
+			},
+			{
+				command:    []string{"uname.exe", "-o"},
+				workingDir: "%CONTAINER_SANDBOX_MOUNT_POINT%\\bin\\",
+			},
+			{
+				command: []string{"powershell", "cmd.exe", "/ver"},
+			},
+			{
+				command: []string{"powershell", "c:/Windows/System32/cmd.exe", "/c", "ver"},
+			},
+			{
+				command: []string{"powershell", "c:\\Windows\\System32/cmd.exe", "/c", "ver"},
+			},
+			{
+				command: []string{"powershell", "%CONTAINER_SANDBOX_MOUNT_POINT%\\bin\\uname.exe", "-o"},
+			},
+			{
+				command: []string{"powershell", "$env:CONTAINER_SANDBOX_MOUNT_POINT\\bin\\uname.exe", "-o"},
+			},
+			{
+				command: []string{"powershell", "%CONTAINER_SANDBOX_MOUNT_POINT%/bin/uname.exe", "-o"},
+			},
+			{
+				command: []string{"powershell", "$env:CONTAINER_SANDBOX_MOUNT_POINT/bin/uname.exe", "-o"},
+			},
+			{
+				command:    []string{"powershell", "bin/uname.exe", "-o"},
+				workingDir: "%CONTAINER_SANDBOX_MOUNT_POINT%",
+			},
+			{
+				command:    []string{"powershell", "bin/uname.exe", "-o"},
+				workingDir: "$env:CONTAINER_SANDBOX_MOUNT_POINT",
+			},
+			{
+				command:    []string{"powershell", "bin\\uname.exe", "-o"},
+				workingDir: "$env:CONTAINER_SANDBOX_MOUNT_POINT",
+			},
+			{
+				command:    []string{"powershell", ".\\uname.exe", "-o"},
+				workingDir: "%CONTAINER_SANDBOX_MOUNT_POINT%/bin",
+			},
+			{
+				command:    []string{"powershell", "./uname.exe", "-o"},
+				workingDir: "%CONTAINER_SANDBOX_MOUNT_POINT%/bin",
+			},
+			{
+				command:    []string{"powershell", "./uname.exe", "-o"},
+				workingDir: "$env:CONTAINER_SANDBOX_MOUNT_POINT\\bin\\",
+			},
+			{
+				command: []string{"%CONTAINER_SANDBOX_MOUNT_POINT%\\bin\\uname.exe"},
+				args:    []string{"-o"},
+			},
+			{
+				command:    []string{"bin\\uname.exe"},
+				args:       []string{"-o"},
+				workingDir: "%CONTAINER_SANDBOX_MOUNT_POINT%",
+			},
+			{
+				command:    []string{"uname.exe"},
+				args:       []string{"-o"},
+				workingDir: "%CONTAINER_SANDBOX_MOUNT_POINT%\\bin",
+			},
+			{
+				command: []string{"cmd.exe"},
+				args:    []string{"/c", "dir", "%CONTAINER_SANDBOX_MOUNT_POINT%\\bin\\uname.exe"},
+			},
+			{
+				command:    []string{"cmd.exe"},
+				args:       []string{"/c", "dir", "bin\\uname.exe"},
+				workingDir: "%CONTAINER_SANDBOX_MOUNT_POINT%",
+			},
+			{
+				command: []string{"powershell"},
+				args:    []string{"Get-ChildItem", "-Path", "$env:CONTAINER_SANDBOX_MOUNT_POINT\\bin\\uname.exe"},
+			},
+			{
+				command:    []string{"powershell"},
+				args:       []string{"Get-ChildItem", "-Path", "bin\\uname.exe"},
+				workingDir: "$env:CONTAINER_SANDBOX_MOUNT_POINT",
+			},
+			{
+				command:    []string{"powershell"},
+				args:       []string{"Get-ChildItem", "-Path", "uname.exe"},
+				workingDir: "$env:CONTAINER_SANDBOX_MOUNT_POINT/bin",
+			},
+		}
+
+		for i, testCase := range tests {
```

Comment:
Added the `[Slow]` tag.

Reply:
I updated the code to batch the test cases and also improve logging. I kept these updates in a separate commit (for now) because it does add some complexity to the code.

@jsturtevant PTAL and let me know if you think it is cleaner to just add a `[Slow]` tag or batch the test cases.

Thanks!

Reply:
It does add some complexity but seems to speed up the tests quite a bit.  Lets go ahead and merge this and see how it works along other tests for the next week.